lesson6.js

```diff
@@ -1,7 +1,13 @@
 const ul = document.getElementById('js-list');
+const loadingTime = document.createElement('img');
+loadingTime.src = "loading-circle.gif";
+loadingTime.id = 'loading-img'
+ul.appendChild(loadingTime);
+const loading = document.getElementById('loading-img');
+
 const info = [
-  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
-  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+  {to: "bookmark.html", img: "bookmark-icon.png", alt:"画像1", text: "ブックマーク"},
+  {to: "message.html", img: "mail-icon.png", alt:"画像2", text: "メッセージ"}
   ];
 const fragment = document.createDocumentFragment();
```

Comment:
IMO
Instead of declaring them globally, it is better to declare them where they will be used, which will reduce bugs.

Reply:
The `const fragment = document.createDocumentFragment();` in line 12 could also be declared here.

Reply:
Thank you for your advice. I tried to describe the associative array in "makeLinkList ()".